src/Resources/Browser.php

```diff
@@ -4,15 +4,15 @@
 
 /**
  * @method mixed|null process()
- * @method \Nesk\Puphpeteer\Resources\BrowserContext createIncognitoBrowserContext()
- * @method \Nesk\Puphpeteer\Resources\BrowserContext[] browserContexts()
- * @method \Nesk\Puphpeteer\Resources\BrowserContext defaultBrowserContext()
+ * @method BrowserContext createIncognitoBrowserContext()
+ * @method BrowserContext[] browserContexts()
+ * @method BrowserContext defaultBrowserContext()
```

Comment:
I alsways use simple FQCN, because its much easier to read. And i think IDE add relative class names if its same namespace by default. 

Problems can occure if some classes will move to other folder. But i dont think it will happen. 

Anyway prefix controlled by one string in console command. And if there will be different namespaces it will cause troubles too. So i suggest to solve problems only if they arrive.

Reply:
> https://github.com/rialto-php/puphpeteer/blob/03ee95752181c948c5b3f4dedbfcdcfbe082bd5e/src/Puppeteer.php#L16-L17
> 
> This is not valid, the proper references should be:
> 
> ```diff
> -  * @method Browser connect(array $options) 
> -  * @method Browser launch(array $options = []) 
> +  * @method Resources\Browser connect(array $options) 
> +  * @method Resources\Browser launch(array $options = []) 
> ```
> 

This is valid, because we have: 
https://github.com/rialto-php/puphpeteer/blob/03ee95752181c948c5b3f4dedbfcdcfbe082bd5e/src/Puppeteer.php#L5